jboss-eap-cd-openshift-launch/added/launch/messaging.sh

```diff
@@ -517,10 +517,18 @@ function inject_brokers() {
     defaultJms="jms-connection-factory=\"$defaultJmsConnectionFactoryJndi\""
   fi
 
-  # new format
-  sed -i "s|jms-connection-factory=\"##DEFAULT_JMS##\"|${defaultJms}|" $CONFIG_FILE
-  # legacy format, bare ##DEFAULT_JMS##
-  sed -i "s|##DEFAULT_JMS##|${defaultJms}|" $CONFIG_FILE
+  if [ "$REMOTE_AMQ_BROKER" = "true" ] || [ -n "${destinations}" ] || [ "x${DISABLE_EMBEDDED_JMS_BROKER}" != "xtrue" ]
```

Comment:
AFAICT the destinations var is not set at this point.

Reply:
ok